app/src/org/commcare/fragments/BaseFragment.kt

```diff
@@ -0,0 +1,69 @@
+package org.commcare.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.fragment.app.Fragment
+import androidx.viewbinding.ViewBinding
+import org.commcare.dalvik.databinding.LoadingBinding
+import org.commcare.interfaces.BaseView
+
+abstract class BaseFragment<B : ViewBinding> : Fragment(), BaseView {
```

Comment:
can we rename to `ConnectBaseFragment`  and `ConnectBaseView` respectively as these are only for Connect classes and we want to maintain a strong nomenclature separation between Connect and CommCare code given our package structure is quite messed up here.